### PR TITLE
Docs for API4: GET /files/{file_id}/link

### PR DESCRIPTION
**Pull request by saturninoabril**
This is documentation for APIv4: GET /files/{file_id}/link

### CODE REVIEW

File: source/v4/files.yaml [hunk 1]
```diff
@@ -104,3 +104,31 @@
...
+        - name: file_id
+          in: path
+          description: The ID of the file to get a link for
+          required: true
+          type: string
+      responses:
+        '200':
+          description: A publicly accessible link to the given file
+          schema:
+            type: string
```
Comment (jwilander) [inline source/v4/files.yaml]: This should actually be
```
schema:
  properties:
    link:
      type: string
```
since it's a JSON object that's returned

Comment (saturninoabril) [inline source/v4/files.yaml]: noted. i've updated the doc. thanks

**Comment by jwilander** [comment]
Needs a rebase, then we're ready to merge

**Comment by saturninoabril** [comment]
ok, done rebase. thanks

**Pull request merged**